diffkemp/simpll/Config.h

```diff
@@ -51,13 +64,15 @@ class Config {
     std::string SecondOutFile;
     // Cache file directory.
     std::string CacheDir;
-    // Path to LLVM IR pattern configuration.
-    std::string PatternConfigPath;
+    // Path to custom LLVM IR differential pattern configuration.
+    std::string CustomPatternConfigPath;
+
+    // The following structure specifies which built-in patterns
+    // should be treated as semantically equal.
+    struct BuiltinPatterns BIPatterns;
```

Comment:
It could be just me but I would prefer just `Patterns`. It's not obvious what `BIPatterns` stands for anyways, so you still have to look up the type (`BuiltinPatterns`) if you're new to this.